examples/thorchain/helper.go

```diff
@@ -23,8 +22,22 @@ func GetEthAddressFromStdout(stdout string) (string, error) {
 	return matches[1], nil
 }
 
-func sendFunds(ctx context.Context, keyName string, toAddr string, amount ibc.WalletAmount, val0 *cosmos.ChainNode) {
-	memo := strings.Repeat("Hello World ", 10)
-	command := []string{"bank", "send", keyName, toAddr, fmt.Sprintf("%s%s", amount.Amount.String(), amount.Denom), "--note", memo}
-	_, _, _ = val0.Exec(ctx, val0.TxCommand(keyName, command...), val0.Chain.Config().Env)
+// We are running many nodes, using many resources. This function is similar to
+// testutils.WaitForBlocks(), but does not hammer calls as fast as possible.
+func NiceWaitForBlocks(ctx context.Context, delta int64, chain ibc.Chain) error {
```

Comment:
nit: I like the idea for this function, but I think adding a time.Duration in the function params would be better suited so its more general? or maybe just a better name, as `nice` in what context to a user? Leaving that to scope for a future PR if desired.

`WaitForBlocksWithDelay(ctx context.Context, delta int64, chain ibc.Chain, searchDelay time.Duration)`